docs/generated/packages/expo/executors/sync-deps.json

```diff
@@ -22,6 +22,11 @@
         "items": { "type": "string" },
         "default": [],
         "description": "An array of npm packages to exclude."
+      },
+      "all": {
+        "type": "boolean",
+        "description": "Sync all dependencies in workspace's package.json.",
```

Comment:
Should explain what this will do exactly. Does it update the root `package.json`? When would you use this option?